### ISSUE TITLE
Similar faces

### ISSUE THREAD

**Issue by Allison2324** [felixrosberg/FaceDancer#44]
Hello, I need advice.

I have two photos of very similar people or the same person but different photos. And in these cases, the algorithm does not work, as if there is some kind of face similarity threshold that does not allow replacing the face in such cases. How does it work?

**Comment by joe-zxh** [comment]
This issue has been mentioned in https://github.com/felixrosberg/FaceDancer/issues/23

It might be the problem of the training dataset

**Comment by felixrosberg** [comment]
Yes, see above link. I suspect the model internally learns to (with error) detect same or not between the two faces.
